.travis.yml

```diff
@@ -32,5 +35,6 @@ install:
  - go get -u github.com/jteeuwen/go-bindata/...
  - yarn
 script:
+ - docker build .
```

Comment:
once docker is building this we can remove `before_install` and `install` scripts and instead use the Docker image to run the test as it is documented here:
https://docs.travis-ci.com/user/build-stages/share-docker-image/

@ry 
It requires publishing to Docker Hub which requires setting username/password for Docker Hub in the TravisCI configurations.

Reply:
That'd be great. You'll include v8worker2 and the compiled V8 in the image?
Are you able to test with travis in your own branch? That would be preferable. Once it's working I can publish it for ry/deno

Reply:
> You'll include v8worker2 and the compiled V8 in the image?
That's the goal. I'll work towards that.

Right now I have a very poor internet connection which doesn't let me download large files so I can't test this locally. Tomorrow when I go back home I'll finish this up. I did  "Allow edits from maintainers." in this PR so feel free to push to this branch if you want

Reply:
@mohsen1 

I recommend build the project docker image on vps and transfer images via scp to your development machine.

If you’d  like to build the image on your development machine ,  you can use  **arg** instruction and env vars to update source, or pull images via proxy.

At last, i recommend use **docker-compose exec** to excute some cmds instead of write some entrypoint script in dockerfile, better maintenance.

good luck.